test/db/anal/vars

```diff
@@ -170,3 +170,122 @@ var int64_t var_18h @ rbp+0x18
 var int64_t var_bp_20h @ rbp+0x20
 EOF
 RUN
+
+NAME=Variable access with misc registers (x86)
+FILE=-
+ARGS=-a x86 -b 64
+CMDS=<<EOF
+e asm.flags=false
+e anal.vars.stackname = true
+afc=ms
+wx 488bc448895808488970104889781841574881ecb00000008364242000488d48884c8d9c24b0000000498b5b10498b7318498b7b20498be3415fc3
+af
+aaef
+afvx
+pdf
+EOF
+EXPECT=<<EOF
+afvR
+   var_20h  0x29
+   var_18h  0x2d
+   var_10h  0x31
+   var_a0h  0x1d
+   var_c0h  0x18
+   var_30h  0x21
+afvW
+   var_20h  0x3
+   var_18h  0x7
+   var_10h  0xb
+   var_a0h
+   var_c0h  0x18
+   var_30h
+/ 59: fcn.00000000 ();
+|           ; var int64_t var_a0h @ rbp-0xa0
+|           ; var int64_t var_20h @ rbp-0x20
```

Comment:
these values with `rbp` look wrong, as `rbp` is nowhere in this function

Reply:
@ret2libc changed it to `rsp`, but I think it would be better if this showed a list of regs+offsets used to access the variable